utils/src/test/kotlin/DeclaredLicenseProcessorTest.kt

```diff
@@ -26,9 +26,11 @@ import org.ossreviewtoolkit.spdx.SpdxLicense
 import org.ossreviewtoolkit.spdx.SpdxLicenseAliasMapping
 import org.ossreviewtoolkit.spdx.SpdxLicenseIdExpression
 
+import io.kotest.core.spec.style.StringSpec
```

Comment:
Nit: While at it, you could also make this block go before the `org` imports.

Reply:
Done.